src/Azure.Functions.ArtifactAssembler/PipelineHelpers/testVsArtifacts.ps1

```diff
@@ -0,0 +1,45 @@
+param (
```

Comment:
What process/where are we invoking this powershell script? Is it in this PR?

Reply:
Nope it's getting invoked in the consolidated pipeline. I will link a sample run in the PR description